.github/workflows/reviewdog-yamllint.yml

```diff
@@ -0,0 +1,19 @@
+---
+name: ReviewDog yamllint
+on:
+  pull_equest:
+    paths:
+      - '**.yml'
+      - '**.yaml'
```

Comment:
add `python.d` config files?

Reply:
Oh yes, I forgot we have config files that are YAML but do not end in a proper YAML extension...

Reply:
Looks like it’s not properly matching the GHA workflows either...